src/Features/Core/Portable/ExtractMethod/OperationStatus_Statics.cs

```diff
@@ -9,14 +9,14 @@ namespace Microsoft.CodeAnalysis.ExtractMethod
     internal partial class OperationStatus
     {
         public static readonly OperationStatus Succeeded = new(OperationStatusFlag.Succeeded, reason: null);
-        public static readonly OperationStatus FailedWithUnknownReason = new(OperationStatusFlag.None, reason: FeaturesResources.Unknown_error_occurred);
-        public static readonly OperationStatus OverlapsHiddenPosition = new(OperationStatusFlag.None, FeaturesResources.generated_code_is_overlapping_with_hidden_portion_of_the_code);
-        public static readonly OperationStatus NoValidLocationToInsertMethodCall = new(OperationStatusFlag.None, FeaturesResources.No_valid_location_to_insert_method_call);
+        public static readonly OperationStatus FailedWithUnknownReason = new(OperationStatusFlag.Failed, reason: FeaturesResources.Unknown_error_occurred);
+        public static readonly OperationStatus OverlapsHiddenPosition = new(OperationStatusFlag.Failed, FeaturesResources.generated_code_is_overlapping_with_hidden_portion_of_the_code);
+        public static readonly OperationStatus NoValidLocationToInsertMethodCall = new(OperationStatusFlag.Failed, FeaturesResources.No_valid_location_to_insert_method_call);
 
-        public static readonly OperationStatus NoActiveStatement = new(OperationStatusFlag.BestEffort, FeaturesResources.The_selection_contains_no_active_statement);
-        public static readonly OperationStatus ErrorOrUnknownType = new(OperationStatusFlag.BestEffort, FeaturesResources.The_selection_contains_an_error_or_unknown_type);
-        public static readonly OperationStatus UnsafeAddressTaken = new(OperationStatusFlag.BestEffort, FeaturesResources.The_address_of_a_variable_is_used_inside_the_selected_code);
-        public static readonly OperationStatus LocalFunctionCallWithoutDeclaration = new(OperationStatusFlag.BestEffort, FeaturesResources.The_selection_contains_a_local_function_call_without_its_declaration);
+        public static readonly OperationStatus NoActiveStatement = new(OperationStatusFlag.Failed, FeaturesResources.The_selection_contains_no_active_statement);
+        public static readonly OperationStatus ErrorOrUnknownType = new(OperationStatusFlag.Succeeded, FeaturesResources.The_selection_contains_an_error_or_unknown_type);
+        public static readonly OperationStatus UnsafeAddressTaken = new(OperationStatusFlag.Succeeded, FeaturesResources.The_address_of_a_variable_is_used_inside_the_selected_code);
+        public static readonly OperationStatus LocalFunctionCallWithoutDeclaration = new(OperationStatusFlag.Failed, FeaturesResources.The_selection_contains_a_local_function_call_without_its_declaration);
```

Comment:
this is only checked to determine if extract method should be suppressed and we shoudl only show 'extract local function'.  it seems reasonable to me (for now) to continue that suppression.  So if you select a local function (and not its declaration), the code action only offers 'extract local function'.